### ISSUE TITLE
hotspots with more than a simle text

### ISSUE THREAD

**Issue by MiMoWe** [mpetroff/pannellum#479]
Hi,
is it possible to have more than a simple text in a hotspot-bubble? for example a formated text, a small image and a link ...
Thanks

**Comment by DanielBiegler** [comment]
Yes, it's just a div. See [this](https://github.com/mpetroff/pannellum/blob/35edd9f5f7d7900515ee77d155c7d9a7683203c8/src/js/pannellum.js#L1599) to get a feel for it.

**Comment by MiMoWe** [comment]
But in this function I can see "escapeHTML(hs.text)". It seems I can't use html on the "standard-way".

**Comment by alketii** [comment]
What about custom hotspots ?

```js
function hotspot(hotSpotDiv, args) {
  hotSpotDiv.classList.add('custom-hotspot');
  let span = document.createElement('span');
  let img = document.createElement('img');
  img.src = 'img/hotspot.png';
  span.innerHTML = '<p>Hello world!</p>';
  hotSpotDiv.appendChild(img);
  hotSpotDiv.appendChild(span);
}
```

**Comment by DanielBiegler** [comment]
Yeah, I meant that you'd have to overwrite it yourself like @alketii did.

**Comment by MiMoWe** [comment]
Thank you, know it works like I need :-)

**Comment by mpetroff** [comment]
When instantiating Pannellum through the API, the only thing the `escapeHTML` function does is convert `\n` to `<br>`, so standard HTML can be used. It only escapes HTML when using the standalone viewer, to prevent XSS attacks (or if the `escapeHTML` option is set to `true`).
